relayer/src/services/relayer/optimism.ts

```diff
@@ -76,6 +76,34 @@ export class OptimismRelayerService extends BaseRelayerService {
     return;
   }
 
+  async safeRelayMessage(message: CrossChainMessage): Promise<boolean> {
+    try {
+      // Finalize message via OptimismPortal (it calls the L1CrossDomainMessenger internally)
+      (await this.messenger.finalizeMessage(message)).wait(1);
+    } catch (err) {
+      if (err.message.includes("OptimismPortal: withdrawal has already been finalized")) {
+        // Here, the case was that OptimismPortal marked the withdrawal as finalized but the relay still failed
+        // We need to call the L1CrossDomainMessenger to relay the message for usinstead (not going via the OptimismPortal)
```

Comment:
Typo: `usinstead`

Reply:
`// We need to call the L1CrossDomainMessenger to relay the message for us instead of going via the OptimismPortal`
Is this better?